java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
+  else result = a.toString()
+}
+
+private string stubAnnotiationSimpleValue(Expr value) {
+  result = value.(FieldAccess).getField().getQualifiedName() or
+  result = value.(Literal).toString()
```

Comment:
Would it make sense to directly use `getLiteral()` instead (to which `toString()` delegates)? Because `toString()` makes no guarantees about its result.

Reply:
Done in 56d1dde.